src/v2/Apps/Order/Components/OfferHistoryItem.tsx

```diff
@@ -4,82 +4,83 @@ import {
   StepSummaryItem,
   StepSummaryItemProps,
 } from "v2/Components/StepSummaryItem"
-import React, { HTMLProps } from "react"
+import React from "react"
 import { createFragmentContainer, graphql } from "react-relay"
 import { RevealButton } from "./RevealButton"
 
-const OfferHistoryItem: React.SFC<{
-  order: OfferHistoryItem_order
-} & StepSummaryItemProps> = ({
-  order: { totalListPrice, lastOffer, offers },
-  ...others
-}) => {
-    const previousOffers = offers.edges.filter(
-      ({ node: { internalID } }) => internalID !== lastOffer.internalID
-    )
+const OfferHistoryItem: React.SFC<
+  {
+    order: OfferHistoryItem_order
+  } & StepSummaryItemProps
+> = ({ order: { totalListPrice, lastOffer, offers }, ...others }) => {
+  const previousOffers = offers.edges.filter(
+    ({ node: { internalID } }) => internalID !== lastOffer.internalID
+  )
 
-    return (
-      <StepSummaryItem {...others}>
-        <Row>
+  return (
+    <StepSummaryItem {...others}>
+      <Row>
+        <Serif size={["2", "3"]} color="black100" weight="semibold">
+          {lastOffer.fromParticipant === "SELLER"
+            ? "Seller's offer"
+            : "Your offer"}
+        </Serif>
+        <Serif size={["2", "3"]} color="black100">
+          {lastOffer.amount}
+        </Serif>
+      </Row>
+      <Row>
+        <div />
+        <Sans size="2" color="black60">
+          List price: {totalListPrice}
+        </Sans>
+      </Row>
+      {lastOffer.note && (
+        <>
+          <Spacer mb={2} />
           <Serif size={["2", "3"]} color="black100" weight="semibold">
             {lastOffer.fromParticipant === "SELLER"
-              ? "Seller's offer"
-              : "Your offer"}
+              ? "Seller's note"
+              : "Your note"}
```

Comment:
This diff seems to be freaking out a bit. Looks like the related change is up top. Weird.

Reply:
I rebased from master, not sure